docker/ansible/README.md

```diff
@@ -0,0 +1,24 @@
+This folder contains the deployment configuration for Viame-Web
+
+# Requirements
+The server must have the following things:
+- The repository cloned to `/home/viame`
+- Ansible
+- The Ansible [Docker Compose](https://docs.ansible.com/ansible/latest/modules/docker_compose_module.html) module
```

Comment:
Typical ansible deployments involve running `ansible-playbook` to ssh into a server for provisioning.  I understand why that's not possible in this case, but I think you should explain the limitations and resulting architecture here, because these requirements would confuse a new developer.

Something to the effect of: "You have to run ansible with a systemd timer on the target server because it isn't accessible via SSH, so traditional CI/CD wont' work.  Also, this project's uptime requirements prohibit any kind of continuous deployment, and we have instead opted for scheduled overnight depolyments."

Reply:
ea4c365